services/local-notifications/core.go

```diff
@@ -475,10 +475,11 @@ func (s *Service) IsStarted() bool {
 }
 
 func SendMessageNotifications(mnb []protocol.MessageNotificationBody) {
+	log.Info("sending message notification", "notification", mnb)
 	var ns []*Notification
 	for _, n := range mnb {
 		ns = append(ns, &Notification{
-			Body:     n,
+			Body:     &n,
 			BodyType: TypeMessage,
```

Comment:
Here you are taking the address of a temporary variable, and it won't work.
You need to take the address from the array:

```
for i, n := range mnb {
   ...
   Body: &ns[i],
  ... 
}
```
Though not sure the type is the right one.

Reply:
well I went another way, can you check it please?

Reply:
probalby is best to have some consistency, either we always pass a pointer or never, so I'd say we either modify the above to have an actual object, or pass a pointer in yours instead of the object